File: Day 1/Python/TaskOne.py


Score_Counter = 0
sum_Of_Scores = 0

while Score_Counter < 10:

	score = int(input("Enter a score: "))

	sum_Of_Scores += score;

	Score_Counter = Score_Counter + 1


print(f'The sum of scores collected is {sum_Of_Scores}');

File: Day 1/Java/TaskOne.java
import java.util.Scanner;
public class TaskOne{
public static void main(String[] args){

Scanner input = new Scanner(System.in);

int ScoreCounter = 0;
int sumOfScores = 0;

while (ScoreCounter < 10){



System.out.print("Enter a score: ");
int score = input.nextInt();

if (score == -1){
break;
}

sumOfScores += score;

ScoreCounter++;
}

System.out.print("The sum of scores collected is "+ sumOfScores);


}
}